lib/plugins/orionPlugin.js

```diff
@@ -38,23 +38,27 @@ var sax = require('sax'),
 function translateAction(originalAction) {
```

Comment:
A couple of entries in CHANGES_NEXT_RELEASE about the changes in this PR should be added. Suggestion:

```
- Support REPLACE (NGSIv1) and replace (NGSIv2) as action type for ContextBroker requests (#422)
```

Reply:
New suggestion: "Support REPLACE (NGSIv1), replace (NGSIv2) and appendStrict (NGSIv2) as action type for ContextBroker requests (#422)."

Commit: 2ac7b34